statsmodels/tsa/vector_ar/util.py

```diff
@@ -191,7 +191,7 @@ def acf_to_acorr(acf):
     return acf / np.sqrt(np.outer(diag, diag))
 
 
-def varsim(coefs, intercept, sig_u, steps=100, initial_values=None, seed=None):
+def varsim(coefs, intercept, sig_u, steps=100, initial_values=None, seed=None, n_sim=1):
```

Comment:
Can we use `nsimulations` which is the same as `SARIMAX`

use `nsimulations=None`.

`None` means 1 simulation with shape (steps, neqs).
Any integer value means (nsimulations, steps, neqs)